help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,250 @@
+
+## Activate segments to this destination {#activate}
+
+Read [Create destinations and activate data](https://experienceleague.adobe.com/docs/platform-learn/tutorials/destinations/create-destinations-and-activate-data.html?lang=en) for generic instructions on activating audience segments to this destination. See below for the screengrabs corresponding to this destination. 
+
+* Within the AEP console, navigate to *Destinations*.
+
+* Click the *Catalog* tab and search for *Sendgrid*. Then click *Activate Segments*.
+
+![](../../assets/catalog/email-marketing/sendgrid/02-catalog.jpg)
+
+* You will be shown a wizard where the first screen is to choose a destination. For setting up this connector, let us create a new destination by clicking on *Configure new destination*.
+
+![](../../assets/catalog/email-marketing/sendgrid/03.jpg)
+
+* Select the *New Account* option and fill in the *Bearer Token* value which is the Sendgrid *API Key* value we had previously copied within [prerequisities].(#prerequisites-prerequisites)
+
+![](../../assets/catalog/email-marketing/sendgrid/04.jpg)
+
+* Upon clicking *Connect to destination*, you will be shown the next [step](#connection-parameters-parameters) to fill in additional information fields.
+
+### Connection parameters {#parameters}
+
+While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in future.
+*  **[!UICONTROL Description]**: An optional description that will help you identify this destination in future.
+
+![](../../assets/catalog/email-marketing/sendgrid/05.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/06.jpg)
+
+## Connect to destination {#connect}
+
+* To connect to this destination, the generic steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) will provide guidance. See below for the screengrabs corresponding to this destination. 
+
+![](../../assets/catalog/email-marketing/sendgrid/07.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/08.png)
+
+* You will be navigated to the list of destinations.
+
+![](../../assets/catalog/email-marketing/sendgrid/09.jpg)
+
+* Select the newly created destination and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/10.jpg)
+
+* Select the *Segment* *(created earlier in [prerequisites](#prerequisites-prerequisites))* and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/11.jpg)
+
+* Now map the *Source* and *Schema* *(created earlier in [prerequisites](#prerequisites-prerequisites))* fields.
```

Comment:
consider rephrasing to: in this step, you need to map XDM fields (identities and attributes) to external fields accepted by the Sendgrid destination.

Reply:
Based on a prior comment we have removed the block containing this content as its covered within the default documentation.